deps/build.jl

```diff
@@ -0,0 +1,33 @@
+if !haskey(ENV, "MKLROOT")
+    error("please set the environmental variable MKLROOT to build MKLSparse.jl")
+end
+
+if Base.USE_BLAS64
+    libmkl_path = joinpath(ENV["MKLROOT"], "lib", "intel64", "libmkl_rt")
```

Comment:
Cool, that works. Didn't think it would be found. I can get rid of all the build stuff then